control_panel_api/services.py

```diff
@@ -14,30 +11,13 @@ def _policy_name(bucket_name, readwrite=False):
     return "{}-{}".format(bucket_name, READWRITE if readwrite else READONLY)
 
 
-def _app_role_name(app_slug):
-    return "{}_app_{}".format(settings.ENV, app_slug)
-
-
 def _policy_arn(bucket_name, readwrite=False):
```

Comment:
Potentially we may have `aws_policy_arn()` in the `S3Bucket` model.
This, IMHO, is very related to an S3Bucket as this is the ARN for policies who determine the access to it.

Reply:
Mmmm, but with the limitation of not being able to pass models instances to functions in `service` I guess it's not possible.

Maybe when we'll have `AWS*` "models" which incapsulates models this would not be a problem.

Reply:
Yeh this area needs another look.